pkg/controller/clusterclaim/clusterclaim_controller_test.go

```diff
@@ -536,6 +620,12 @@ func TestReconcileClusterClaim(t *testing.T) {
 					Reason:  "Resuming",
 					Message: "Waiting for cluster to be running",
 				},
+				{
```

Comment:
`expectedConditions: []hivev1.ClusterClaimCondition{
				{
					Type:    hivev1.ClusterClaimPendingCondition,
					Status:  corev1.ConditionFalse,
					Reason:  "ClusterClaimed",
					Message: "Cluster claimed",
				},
				{
					Type:    hivev1.ClusterRunningCondition,
					Status:  corev1.ConditionFalse,
					Reason:  "Resuming",
					Message: "Waiting for cluster to be running",
				},
				{
					Type:    hivev1.ClusterClaimClusterDeletedCondition,
					Status:  corev1.ConditionFalse,
					Reason:  "ClusterNotDeleted",
					Message: "Assigned cluster has not been deleted",
				},
			},`
This set of expected condition status are repeated many times, is it possible to break it out in a separate function for happy path?

Reply:
✓